torch/csrc/jit/ir.h

```diff
@@ -93,7 +93,7 @@ struct Scope {
   }
   Scope* push(Symbol name) {
     Scope* newScope = new Scope(this, name);
-    children_.insert(std::unique_ptr<Scope>(newScope));
+    children_.push_back(std::unique_ptr<Scope>(newScope));
```

Comment:
In general the problem with this line is that it can leak memory if `push_back` fails (e.g. because an allocation fails). It would be better to do this:
```cpp
children_.push_back(std::unique_ptr<Scope>(new Scope(this, name)));
return children_.back().get();
```